app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
```

Comment:
The use of a backslash for line continuation is unnecessary here. Python allows breaking lines inside parentheses, brackets, and braces without using a backslash.